### ISSUE TITLE
Broken support for WebPack's `enhanced-resolve` due to incorrect order of "default" condition in `package.json` "exports"

### ISSUE THREAD

**Issue by danielweck** [valeriangalliat/fetch-cookie#72]
Culprit:
https://github.com/valeriangalliat/fetch-cookie/blob/b144298b45c4c8e45a38bb9b8cdb176c785a09ad/package.json#L7-L20

Error:
"[webpack-cli] ModuleNotFoundError: Module not found: Error: Default condition should be last one"

Origin:
https://github.com/webpack/enhanced-resolve/blob/ddc96f8e738690166e7de77ea09c9e5aff52bb1b/lib/util/entrypoints.js#L453-L457
```javascript
			if (i !== last) {
				if (condition === "default") {
					throw new Error("Default condition should be last one");
				}
			} 
```

**Comment by valeriangalliat** [comment]
Oh wow thanks for reporting. Fixed in 2.0.5!

**Comment by danielweck** [comment]
Thank you for your quick response! :)
Working fine now.
